cou/cli.py

```diff
@@ -217,23 +214,21 @@ async def _run_command(args: CLIargs) -> None:
     :param args: CLI arguments
     :type args: CLIargs
     """
-    match args.command:
-        case "plan":
-            await get_upgrade_plan(args)
-        case "upgrade":
-            await run_upgrade(args)
+    model = await get_model(args)
+    cloud_upgrade_plan = await analyze_and_generate_plan(model, args)
```

Comment:
We could add a comment here about why `"plan"` isn't explicitly handled here any more.

Reply:
removed